nailgun/entity_mixins.py

```diff
@@ -839,7 +847,10 @@ def create(self, create_missing=None):
             on the current object.
 
         """
-        return self.read(attrs=self.create_json(create_missing))
+        signals.pre_create.send(self, create_missing=create_missing)
+        entity = self.read(attrs=self.create_json(create_missing))
+        signals.post_create.send(self, entity=entity)
+        return entity
```

Comment:
This is not a **read** this is **create** take a look https://github.com/rochacbruno/nailgun/blob/add_signals/nailgun/entity_mixins.py#L824-L850 explanation on docstring (looks like **.read** actually creates with that attrs create_missing.